tuxemon/event/conditions/check_evolution.py

```diff
@@ -41,14 +41,14 @@ def test(self, session: Session, condition: MapCondition) -> bool:
             logger.error(f"{_character} not found")
             return False
 
-        self.character = character
-        self.client = session.client
+        context = {"map_inside": session.client.map_inside, "use_item": False}
 
         evolving_monsters = []
         for monster in character.monsters:
             if monster.evolutions:
                 for evolution in monster.evolutions:
-                    if self.can_evolve(monster, evolution):
+                    if can_evolve(monster, evolution, context):
+                        print(monster.name)
```

Comment:
remove print statement?

Reply:
you're right, I forgot it, I'm going to remove it